lib/full_url_for.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const { parse } = require('url');
+const encodeURL = require('./encode_url');
+
+function fullUrlForHelper(path = '/') {
+  if (path.startsWith('//')) return path;
+  const { config } = this;
+  const data = parse(path);
+
+  // Exit if this is an external path
+  if (data.protocol) return path;
+
+  // Prepend root path
```

Comment:
I don't think this comment is necessary, the next line is obvious on what it does.

Reply:
Removed.